components/board/genericlinux/board.go

```diff
@@ -273,21 +273,17 @@ func (b *Board) reconfigureInterrupts(newConf *LinuxBoardConfig) error {
 	for _, oldInterrupt := range b.interrupts {
 		if newConfig := findNewDigIntConfig(oldInterrupt, newConf.DigitalInterrupts, b.logger); newConfig == nil {
 			// The old interrupt shouldn't exist any more, but it probably became a GPIO pin.
-			if err := closeInterrupt(oldInterrupt); err != nil {
-				return err // This should never happen, but the linter worries anyway.
+			if err := oldInterrupt.Close(); err != nil {
+				return err
 			}
 			if newGpioConfig, ok := b.gpioMappings[oldInterrupt.config.Pin]; ok {
-				// See gpio.go for createGpioPin.
```

Comment:
This comment should have been removed in https://github.com/viamrobotics/rdk/pull/3972, but I missed it.